docusaurus/docs/protocol/upgrades/upgrade_procedure.md

```diff
@@ -67,42 +70,143 @@ An upgrade transaction includes a [Plan](https://github.com/cosmos/cosmos-sdk/bl
+
+```bash
+poktrolld tx authz exec tools/scripts/upgrades/authz_cancel_upgrade_tx.json --gas=auto --from=pnf
+```
+
 ## Testing the Upgrade
 
 :::warning
-Note that for local testing, `cosmovisor` won't pull the binary from the info field.
+Note that for local testing, `cosmovisor` won't pull the binary from the upgrade Plan's info field.
 :::
 
 ### LocalNet
 
-LocalNet currently does not support `cosmovisor` and automatic upgrades. However, we have provided scripts to facilitate local testing in the `tools/scripts/upgrades` directory:
-
-1. Modify `tools/scripts/upgrades/authz_upgrade_tx_example_v0.0.4_height_30.json` to reflect the name of the upgrade and the height at which it should be scheduled.
-
-2. Check and update the `tools/scripts/upgrades/cosmovisor-start-node.sh` to point to the correct binaries:
-
-   - The old binary should be compiled to work before the upgrade.
-   - The new binary should contain the upgrade logic to be executed immediately after the node is started using the new binary.
-
-3. Run `bash tools/scripts/upgrades/cosmovisor-start-node.sh` to wipe the `~/.poktroll` directory and place binaries in the correct locations.
-
-4. Execute the transaction as shown in [Submitting the upgrade on-chain](#submitting-the-upgrade-on-chain) section above.
+LocalNet **DOES NOT** support `cosmovisor` and automatic upgrades at the moment.
+
+However, **IT IS NOT NEEDED** to simulate and test the upgrade procedure.
+
+#### LocalNet Upgrade Cheat Sheet
+
+For a hypothetical scenario to upgrade from `0.1` to `0.2`:
+
+1. **Stop LocalNet** to prevent interference. Pull the `poktroll` repo into two separate directories. Let's name them `old` and `new`. It is recommended to open at least two tabs/shell panels in each directory for easier switching between directories.
+
+2. **(`old` repo)** - Check out the old version. For the test to be accurate, we need to upgrade from the correct version.
+   ```bash
+   git checkout v0.1
+   ```
+
+3. **(`new` repo)** 
+   ```bash
+   git checkout -b branch_to_test
+   ```
+   Replace `branch_to_test` with the actual branch you want to test. **Note:** This branch should have an upgrade implemented - [Implementing the Upgrade](#implementing-the-upgrade). Here, the upgrade should be named `v0.2`.
+
+4. **(BOTH repos)** - We'll use binaries from both versions - old and new.
+   ```bash
+   make go_develop ignite_release ignite_release_extract_binaries
+   ```
+   :::note
+   The binary produced by these commands in the old repo should result in the same binary as it was downloaded from [production releases](https://github.com/pokt-network/poktroll/releases), however you might consider using 
+   :::
+
+5. **(`old` repo)** - Clean up and generate an empty genesis using the old version.
+   ```bash
+   rm -rf ~/.poktroll && ./release_binaries/poktroll_darwin_arm64 comet unsafe-reset-all && make localnet_regenesis
```

Comment:
make target?

Reply:
With these commands, I want a developer to 100% understand what they are doing, checking all arguments.